infra/gcp/clusters/k8s-infra-prow-build/prow-build/main.tf

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2020 The Kubernetes Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+This file defines:
+- GCP Project k8s-infra-prow-build to hold a prow build cluster
+- GCP Service Account for k8s-infra-prow-build pods (bound via workload identity to a KSA of the same name)
+- GCP Service Account for boskos-janitor (bound via workload identity to a KSA of the same name)
+- GKE cluster configuration for prow-build
+- GKE nodepool configuration for prow-build
+*/
+
+locals {
+  project_id              = "k8s-infra-prow-build"
+  cluster_name            = "prow-build"           // The name of the cluster defined in this file
+  cluster_location        = "us-central1"          // The GCP location (region or zone) where the cluster should be created
+  bigquery_location       = "US"                   // The bigquery specific location where the dataset should be created
+  pod_namespace           = "test-pods"            // MUST match whatever prow is configured to use when it schedules to this cluster
```

Comment:
You can, but I can't change it right now.  We have to match https://github.com/kubernetes/test-infra/blob/master/config/prow/config.yaml#L73, which has been set to this since epoch.  Changing that requires changing all other build clusters hooked up to prow.k8s.io (not just ours) to use something other than `test-pods`.  Eventually the prow team hopes to remove this hardcode, and make the namespace configurable per-cluster, at which point we could use something else.